packages/shorebird_cli/lib/src/cache.dart

```diff
@@ -198,7 +199,10 @@ class AotToolsArtifact extends CachedArtifact {
   String get name => 'aot-tools';
 
   @override
-  List<String> get executables => ['aot-tools'];
+  String get fileName => 'aot-tools.dill';
```

Comment:
Why couldn't we change `name` to be `aot-tools.dill` and avoid introducing a new `fileName`?

Reply:
Because the artifact path is `[name]/[fileName]` (or, in the case of aot-tools, `[name]/[flutterRev]/[fileName]`). We could use fileName for both, but this felt cleaner.